components/building-highlights/building-highlights.component.jsx

```diff
@@ -10,7 +10,11 @@ const BuildingHighlights = () => {
   /**
    * Fill color for the Polygons
    */
-  const [fillColor, setFillColor] = useState("rgba(128, 0, 32, 0.5)");
+  const [fillColor, setFillColor] = useState(null);
```

Comment:
useState() allows you to set a default value, but it does not mean that your value is set

Reply:
Yeah, this is a workaround, since there is a bug with map view where the map needs to be rerendered a second time to apply certain changes